addons/sys_sem70/functions/fnc_clearDisplay.sqf

```diff
@@ -21,4 +21,4 @@
 params ["_display"];
  {
     RADIO_CTRL(_x) ctrlSetText "";
-} forEach [301,302,303,304,305/*,109*/]; // purge.
+} forEach [301, 302, 303, 304, 305]; // purge.
```

Comment:
That should really be a define in `script_component.hpp`. But all radio components could benefit from those.

Reply:
Let's have a complete clean of those in the future and make it common to all radios.